modules/ocf_stats/manifests/prometheus.pp

```diff
@@ -1,8 +1,21 @@
 # prometheus daemon config
 class ocf_stats::prometheus {
+  file {
+    '/usr/local/bin/gen-prometheus-nodes':
+      source => 'puppet:///modules/ocf_stats/prometheus/gen-prometheus-nodes',
+      mode   => '0755';
+  }
+
+  cron { 'gen-prometheus-nodes':
+    command => '/usr/local/bin/gen-prometheus-nodes /var/local/prometheus-nodes.json',
+    user    => 'root',
+    minute  => '03',
```

Comment:
nit: just do '3'.

EDIT: Saw the `refresh_interval` below.
~Also, is this meant to run every 3 minutes? If so you are looking for `*/3`. Just asking because the 3rd minute of every hour seems a little arbitrary~

Reply:
Also, every hour seems a little excessive for how often the nodes are updated. Realistically, how often do these change?

Reply:
I took this from the munin config which does it at the 3rd minute as well. It doesn't really hurt to query LDAP every hour, and that's about how long we can acceptably wait before monitoring starts for a new host, so I think it's fine.